File: src/main/java/at/dietze/quadru/factories/PlayerNameFormatter.java
package at.dietze.quadru.factories;

public class PlayerNameFormatter {

    public static String format(String playerName, String islandName) {
        if (islandName == null || islandName.isEmpty()) {
            islandName = "OBDACHLOS";
        }

        switch (islandName) {
            case "aloria":
                islandName = "§9ALORIA";
                break;
            case "pyroka":
                islandName = "§6PYROKA";
                break;
        }

        return "§8(§7" + islandName + "§8)" + " §7" + playerName + "§r";
    }
}

File: src/main/java/at/dietze/quadru/factories/PlayerNameFormatter.kt
package at.dietze.quadru.factories

object PlayerNameFormatter {
    @JvmStatic
    fun format(playerName: String, islandName: String?): String {
        var islandName = islandName
        if (islandName == null || islandName.isEmpty()) {
            islandName = "OBDACHLOS"
        }

        when (islandName) {
            "aloria" -> islandName = "§9ALORIA"
            "pyroka" -> islandName = "§6PYROKA"
        }

        return "§8(§7$islandName§8) §7$playerName§r"
    }
}
